inc/server/class-prompt-server.php

```diff
@@ -47,6 +47,13 @@ class Prompt_Server {
 	 */
 	public $timeout_transient = 'otter_prompts_timeout';
 
+	/**
+	 * OpenAI Endpoint.
+	 * 
+	 * @var string
+	 */
+	private static $base_url = 'https://api.openai.com/v1/chat/completions';
```

Comment:
Love the idea of `private static` here 🙌 

What you can do now since PHP 7.1 is use private constants since you don't want to edit `base_url` in any scenario. So you could do `private const BASE_URL` and then do `self::BASE_URL`. 🙂

Reply:
Oh ok, I was basing myself on Optimole which I think doesn't go below PHP 7.1. What's the minimum version for Otter blocks?

Reply:
Minimum is 5.6, I suppose we should bump that to 7.1 at some point but currently we support that far back.